tests/riotboot/Makefile

```diff
@@ -21,6 +21,11 @@ QUIET ?= 1
 # Thus default to that (instead of epoch set by makefiles/boot/riotboot.inc.mk).
 APP_VER?=0
 
+# To avoid compilation during the test ensure both slot bin files are always
+# generated and linked this ensures that "BUILD_IN_DOCKER=1 make test" and
```

Comment:
Looks like the `and` at the end of the line is not needed. Otherwise the sentence is a bit weird. Maybe also add some punctuation between `linked` and `this ensures`: a period or a comma.

Reply:
addressed.